peril.settings.json

```diff
@@ -6,17 +6,15 @@
   },
   "rules": {
     "pull_request": "artsy/artsy-danger@org/all-prs.ts",
-    "pull_request.closed": "artsy/artsy-danger@org/closed-prs.ts"
+    "pull_request.closed": "artsy/artsy-danger@org/closed-prs.ts",
+    "issues.opened": "artsy/artsy-danger@danger/new_rfc.ts"
   },
   "repos": {
     "artsy/reaction": {
       "pull_request": "danger/pr.ts"
     },
     "artsy/positron": {
       "pull_request": "dangerfile.ts"
-    },
-    "artsy/artsy-danger": {
-      "issues.opened": "artsy/artsy-danger@danger/new_rfc.ts"
     }
```

Comment:
Moves this spec from just `artsy/artsy-danger` to all repos. Will need to restart our heroku instance to make this change actually happen after it is merged.